### ISSUE TITLE
Branches 

### ISSUE THREAD

**Issue by DarinBeard** [CodeHubApp/CodeHub#407]
I don’t know how people use this for merging when there is no indication what branches are being merged. When I go look at a pull request is does not tell me what branches are affected by the request. I can’t possibly hit merge on it when I don’t know what branch it is going to be merged into. That seems like a critical piece of data to me. I like this app a lot and it will be a wonderful thing that if were added so I could actually use it for what I need it for. Thanks for listening.

**Comment by thedillonb** [comment]
@DarinBeard good point. I'll put that information into the next update. Thanks for pointing this out.
